readme.md

```diff
@@ -153,6 +153,12 @@ antigen bundle sindresorhus/pure
 
 ## FAQ
 
+### My preprompt is missing when I clear the screen with Ctrl+L
+
+Pure doesn't register its custom *clear-screen* widget if it has been previously modified. If you haven't registered your own zle widget with `zle -N clear-screen custom-clear-screen` it might have been done by third-party modules. For example `zsh-syntax-highlighting` and `zsh-history-substring-search` are known to do this and they should for that reason be **the very last thing** in your `.zshrc` (as pointed out in their documentation).
+
+To find out the culprit that is overriding your *clear-screen* widget, you can run the following command: `zle -l | grep clear-screen`.
+
```

Comment:
This section is awesome. Nice work.

Though it's funny how creating a ZSH prompts ends up having to document most of ZSH :p